components/brave_news/browser/resources/Feed.tsx

```diff
@@ -121,23 +123,27 @@ export default function Component({ feed, onViewCountChange, onSessionStart }: P
   // Create intersection observer & relevant state to measure
   // the amount of viewed cards in the session.
   const lastViewedCardCount = React.useRef(0);
+  const lastUsageTime = React.useRef<Date | null>(null);
```

Comment:
Not 100% sure if it's possible but this will also count remounts of the component as new sessions. Not sure about when/if this could happen but disabling/reenabling news will count as a new session, for example.

Reply:
> disabling/reenabling news will count as a new session, for example.

That is indeed the case, should be okay.